app/services/submissions/creation_service.rb

```diff
@@ -12,6 +12,8 @@ def call(params, role)
             application: params[:application],
             version: 1,
           )
+
+          submission.update_columns(last_updated_at: submission.created_at)
```

Comment:
ideally we should pull this from `params[:application]['updated_at']` but that is currenlty only populated for NSM applications... I would push to add it to the PA payload to include it before merging this.

This guards against delays in sidekiq processing affecting the recorded order of submission.